src/main/java/it/gov/pagopa/payhub/pdnd/service/ApiClientConfigService.java

```diff
@@ -0,0 +1,28 @@
+package it.gov.pagopa.payhub.pdnd.service;
+
+import it.gov.pagopa.payhub.pdnd.anpr.connector.AnprApiClientConfig;
+import it.gov.pagopa.payhub.pdnd.config.pdnd.PdndServiceIntegratedConfig;
+import it.gov.pagopa.payhub.pdnd.send.connector.SendApiClientConfig;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ApiClientConfigService {
```

Comment:
Resolver or Retriever
```suggestion
public class ApiClientConfigResolverService {
```